app/src/main/res/values-ru/strings.xml

```diff
@@ -144,7 +147,7 @@
     <string name="pref_title_notification_settings">Уведомления</string>
     <string name="pref_title_edit_notification_settings">Изменить уведомления</string>
     <string name="pref_title_notifications_enabled">Push-уведомления</string>
-	<string name="pref_title_pull_notification_check_interval">Интервал проверки</string>
+    <string name="pref_title_pull_notification_check_interval">Интервал проверки</string>
```

Comment:
We should add that this is about notifications, don't you think?

Reply:
@charlag Good question, Ivan. I think there shouldn't be any changes, because the subtitle says "Уведомления" and the very next option is obviously about notifications, so there shouldn't be any confusion at all.

![image](https://user-images.githubusercontent.com/2780044/32508104-4f296cd2-c3fa-11e7-80f2-79f2a36806cd.png)

But there are some another things I'd like to discuss:
1. The "time intervals" strings (1 minute, 2 minutes...) can't be translated, right?
2. What about changing "Изменить уведомления" to "Настройки уведомлений"? IMO it looks more "standard" for applications.

Reply:
Ah, yes. you're absolutely right, I was on my phone and missed some context, thank you.

1. They can be, please look for "array.xml" in different "values" folders.
2. Yes, I totally agree, some far more natural.

Thank you for your translations, they're pretty good. I am always skeptical about translations (so all my gadgets and services are in English) but yours are natural.